fulllogging.properties

```diff
@@ -0,0 +1,17 @@
+# Global logging properties.
```

Comment:
Unless you are writing in some code to load that properties file, Java is only going to ignore it.